hangman.js

```diff
@@ -1 +1,107 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt('gimme a word:');
+var guesses = [];
+var attempts = word.length + 4;
+var attemptsLeft = attempts;
+var display = '';
+
+myl = 'a';
+place = word.indexOf(myl);
+
+function showDisplay() {
+  for(i = 0; i < word.length; i++) {
+    display += '-';
+  }
+  display;
+}
+function getGuess() {
+  if(display === word) {
+    attemptsLeft = 0;
+    win();
+  } else {
+    while (attemptsLeft > 0) {
+      var guess = prompt('You have ' + attemptsLeft + ' attempts. ' + 'guess a letter:\n' + display);
+      var place = word.indexOf(guess);
+      if (place > -1) {
+          attemptsLeft--;
+          display = display.replace(display[place], guess);
+          getGuess();
+
+      } else if (place < 0) {
+        attemptsLeft--;
+        if(attemtps == 0) {
+          lose();
+        }
+        }
+        getGuess();
+      }
+    }
+}
+
+function win() {
+    var playAgain = prompt('you win! play again? (y/n)');
+    if (playAgain == 'y') {
+    gameOn();
+    } else if (playAgain == 'n') {
+      alert('thanks for playing!');
+      return;
+    }
+}
+  function lose() {
+    var playAgain = prompt('you have no guesses left. play again? (y/n)');
+    if (playAgain == 'y') {
+      attemptsLeft = attempts;
+      gameOn();
+    } else if (playAgain == 'n') {
+      alert('thanks for playing!');
+      return;
+    }
+  }
+
+
+function gameOn() {
+  showDisplay();
+  getGuess();
+}
+
+/*
+gameOn() {
+  build display with missing letters as -'s
```

Comment:
It's definitely okay when you are starting off to comment out code for debugging, but I would encourage you to instead of doing this, just checkout a new branch for experimentation :+1: